tmva/tmva/test/TestRandomGenerator.cxx

```diff
@@ -0,0 +1,101 @@
+
+// ROOT
+#include "TRandom.h"
+#include "TRandom1.h"
+#include "TRandom2.h"
+#include "TRandom3.h"
+#include "TRandomGen.h"
+
+// TMVA
+#include "TMVA/Tools.h"
+
+// Stdlib
+#include <iostream>
+
+using TRandom1StdGen = TMVA::RandomGenerator<TRandom1, UInt_t, kMaxUInt << 2>;
+using TRandom2StdGen = TMVA::RandomGenerator<TRandom2, UInt_t, kMaxUInt << 2>;
+using TRandom3StdGen = TMVA::RandomGenerator<TRandom3>;
+using TRandomMixMaxStdGen = TMVA::RandomGenerator<TRandomMixMax>;
+using TRandomMixMax17StdGen = TMVA::RandomGenerator<TRandomMixMax17>;
+using TRandomMixMax256StdGen = TMVA::RandomGenerator<TRandomMixMax256>;
+
+template <typename URNG>
+void print_urng(URNG urng, int n = 10)
+{
+   std::uniform_int_distribution<UInt_t> dist{2, 5};
+
+   for (int i = 0; i < n; ++i) {
+      std::cout << dist(urng) << ", ";
+   }
+   std::cout << std::endl;
+}
+
+// Test that generation compiles
+void test_print(int n)
```

Comment:
Could you please use Google test? Or is there a special policy for TMVA that we shouldn't use it?

Reply:
It uses gtest now :)